### PR TITLE
Add Simplified Chinese translation of the doctrine

### PR DESCRIPTION
**Pull request by huacnlee**
Original link:
https://ruby-china.org/wiki/the-rails-doctrine

### CODE REVIEW

File: doctrine/zh_cn.html [hunk 1]
```diff
@@ -0,0 +1,309 @@
+---
+layout: doctrine
+title: 信条
+permalink: /doctrine/zh_cn
+---
+
+    <section>
+      <h1 style="margin-bottom: 0px">Rails 信条</h1>
+      <p style="margin-top: 0"><small style="color: grey">David Heinemeier Hansson 撰写于 2016 年 1 月</small></p>
```
Comment (JuanitoFatas) [inline doctrine/zh_cn.html]: Missing translator information here.
See: https://github.com/rails/homepage/blob/7a9b69403826a1f9b57b64dd129fbb3edbac7f94/doctrine/ru.html#L10

Comment (huacnlee) [inline doctrine/zh_cn.html]: No need this.

Comment (JuanitoFatas) [inline doctrine/zh_cn.html]: How about credit the community members of Ruby China?

```html
<p style="margin: 0; width: 340px;">
  <small style="color: grey">By David Heinemeier Hansson 撰写于 2016 年 1 月（<a href="https://ruby-china.org" target="_blank" style="color: #716a6a">Ruby China 社区</a>译）</small><br>
</p>
```

File: doctrine/zh_cn.html [hunk 2]
```diff
@@ -0,0 +1,309 @@
+---
+layout: doctrine
+title: 信条
```
Comment (JuanitoFatas) [inline doctrine/zh_cn.html]: `信条` => `Rails 信条`?

screen shot 2017-08-21 at 18 06 34

File: doctrine/zh_cn.html [hunk 3]
```diff
@@ -0,0 +1,309 @@
...
+permalink: /doctrine/zh_cn
+---
+
+    <section>
```
Comment (JuanitoFatas) [inline doctrine/zh_cn.html]: Missing lang attribute here.

https://github.com/rails/homepage/blob/7a9b69403826a1f9b57b64dd129fbb3edbac7f94/doctrine/ru.html#L7

**Comment by JuanitoFatas** [comment]
👏 💯 

dg-voq0uwaa3tmc

[image credit](https://twitter.com/tenderlove/status/896098931108532225)

**Pull request merged**